Dockerfile

```diff
@@ -86,6 +111,17 @@ RUN wget --quiet -O /tmp/cmake.sh https://cmake.org/files/v3.10/cmake-3.10.1-Lin
   && sh /tmp/cmake.sh --prefix=/usr/local --exclude-subdir --skip-license \
   && rm /tmp/cmake.sh
 
+# Install Conan
+RUN pip install conan==1.0.4
+ENV CONAN_CMAKE_TOOLCHAIN_FILE="${CMAKE_TOOLCHAIN_FILE}" \
+  CONAN_PRINT_RUN_COMMANDS=1 \
+  CC="${WSBU_C_COMPILER}" \
+  CXX="${WSBU_CXX_COMPILER}"
+COPY conan/sitara_profile "${HOME}/.conan/profiles/sitara"
+COPY conan/settings.yml "${HOME}/.conan/settings.yml"
+COPY conan/registry.txt "${HOME}/.conan/registry.txt"
+RUN ln -s "${HOME}/.conan/profiles/sitara" "${HOME}/.conan/profiles/default"
```

Comment:
How do we ensure that these .conan profiles exist? Are they going to be cloned from somewhere or something?